_layouts/default.html

```diff
@@ -11,6 +11,7 @@
             });
         });
     </script>
+    <link rel="stylesheet" href="https://ieee.nitk.ac.in/virtual-expo/assets/css/style.css">
```

Comment:
I assume you're calling this stylesheet only for the tab pills, if that's the case can you move just that CSS to this repository?

Reply:
Yeah, it's called for the tab pills. I will move it to repository.